site/content/en/blog/releases/1.33.0.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "1.33.0 - SDKGracefulTermination To Stable and Returning Metadata on Allocation"
+linktitle: "1.33.0"
+date: "2023-07-05"
+---
+
+This is the 1.33.0 release of Agones.
+
+In this release, we have focused on improving the documentation, stabilized the SDKGracefulTermination feature and also are now returning GameServer annotations and labels on allocation response.
+
+- Stable SDKGracefulTermination: 
+    SDKGracefulTermination feature is promoted to stable. With this advancement, the SDK server will now remain active for the specified `terminationGracePeriodSeconds` until `SDK.Shutdown()` has been called. This ensures a seamless transition during shutdown scenarios and gives you more control over the termination process.
+
+- Enhanced Documentation: 
+	We have taken steps to improve the usability of our [helm documentation](https://github.com/googleforgames/agones/blob/05ab9ca00ece135abae21611a67344777b30dc36/site/content/en/docs/Installation/Install%20Agones/helm.md). The large configuration table has been restructured into multiple sections making it much easier to navigate the Agones services.
```

Comment:
Can you change the link to https://agones.dev/site/docs/installation/install-agones/helm/#configuration?

Reply:
I have modified the helm-configuration link as per your suggestion.